tests/unit/s2n_certificate_extensions_test.c

```diff
@@ -80,7 +80,7 @@ const char tls13_cert_single_ext_hex[] =
     "5156726096fd335e5e67f2dbf102702e608ccae6be"
     "c1fc63a42a99be5c3eb7107c3c54e9b9eb2bd5203b"
     "1c3b84e0a8b2f759409ba3eac9d91d402dcc0cc8f8"
-    "961229ac9187b42b4de1000400050000";
+    "961229ac9187b42b4de100050005000100";
```

Comment:
is this change something to the effect of "length one of value 00" ?

Reply:
It's actually the status request type, which I've set to a value that we ignore (x00). I could also add 4 bytes instead of 1 if I wanted to send a valid oscp request type + zero length ocsp data (x01, x000000), but I don't think that makes the test any more thorough.

Reply:
I added a some comments to explain this hex blob :)